bears/java/CheckstyleBear.py

```diff
@@ -67,31 +63,9 @@ def create_arguments(
             - geosoft - The Java style followed by GeoSoft. More info at
               <http://geosoft.no/development/javastyle.html>
         """
-        checkstyle_configs = self.known_configs(checkstyle_configs)
+        if checkstyle_configs in known_checkstyles:
+            checkstyle_configs = self.download_cached_file(
+                known_checkstyles[checkstyle_configs],
+                checkstyle_configs + ".xml")
 
-        return ('-jar', checkstyle_jar_file, '-c', checkstyle_configs, filename)
-
-    def known_configs(self, key):
-        url = known_checkstyles.get(key, None)
-        if url is None:  # If not known, it must be custom path
-            return key
-
-        filename = os.path.join(self.data_dir, key + "_style.xml")
-        if os.path.exists(filename):
-            return filename
-
-        self.info("Downloading checkstyle configs for {key} from {url}"
-                  .format(key=key, url=url))
-        with urlopen(url) as response, open(filename, 'wb') as out_file:
-            copyfileobj(response, out_file)
-        return filename
-
-    @property
-    def data_dir(self):
-        _data_dir = os.path.abspath(os.path.join(
-            appdirs.user_data_dir('coala-bears', version=VERSION),
-            self.name))
-
-        if not os.path.isdir(_data_dir):  # pragma: no cover
-            os.makedirs(_data_dir)
-        return _data_dir
+        return '-jar', checkstyle_jar_file, '-c', checkstyle_configs, filename
```

Comment:
okay acked it already and not that important, but the diff looks better if you don't remove the braces. Especially the change is removed again in the next commit^^